ddtrace/contrib/influx/patch.py

```diff
@@ -0,0 +1,83 @@
+import influxdb  # version >=5.0
+import wrapt
+from influxdb.exceptions import InfluxDBClientError, InfluxDBServerError
+
+from . import metadata
+
+from ...utils.wrappers import unwrap
+from ...compat import urlencode
+from ...pin import Pin
+from ...ext import http, db, AppTypes
+
+
+DEFAULT_SERVICE = 'influxdb'
```

Comment:
We're changing the way we provide defaults and configuration to use our `config` api.

See our celery integration for how this is being done:

definition:
https://github.com/DataDog/dd-trace-py/blob/9d196d476b85096db07a0f099f5605f0e20b9b49/ddtrace/contrib/celery/patch.py#L10-L14

usage:
https://github.com/DataDog/dd-trace-py/blob/9d196d476b85096db07a0f099f5605f0e20b9b49/ddtrace/contrib/celery/signals.py#L34-L36

this approach allows the integration to be configured without needing to use a `Pin` in the user's source.

Reply:
I think I understand how Config is being used here, and changed accordingly, though I'm not sure whether the `app_name` should be subject to config. The Celery integration leaves it as a constant.

Reply:
Yeah, no worries about the `app_name` this is soon-to-be deprecated. It can be left out of the config.